### PR TITLE
TELCODOCS-1315: Correct PGT/CGU for platform update

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.12, 4.13, 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1315

Link to docs preview: https://65327--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.html#talo-platform-update_ztp-talm

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cnf-topology-aware-lifecycle-manager-platform-update.adoc [hunk 1]
```diff
@@ -57,8 +57,6 @@ spec:
       policyName: "platform-upgrade-prep"
```
Comment (jc-rh) [inline modules/cnf-topology-aware-lifecycle-manager-platform-update.adoc:57]: This is no longer needed in the prep policy.

Comment (amolnar-rh) [reply modules/cnf-topology-aware-lifecycle-manager-platform-update.adoc]: @jc-rh This is what you mean? [Diff only](https://github.com/openshift/openshift-docs/pull/65327/commits/4aa4ecaffc001a734d3a082af55a32276fc5fa8d)

Comment (jc-rh) [reply modules/cnf-topology-aware-lifecycle-manager-platform-update.adoc]: Yes

**Review by jeana-redhat — COMMENTED** [review]
LGTM, just be sure to squash commits before merge

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by josclark42** [comment]
LGTM from QE

**Pull request merged**